Packages/io.chainsafe.web3-unity/Editor/ReadWriteProvider.cs

```diff
@@ -0,0 +1,42 @@
+using System.IO;
+using System.Linq;
+using ChainSafe.Gaming.UnityPackage.Connection;
+using UnityEditor;
+using UnityEngine;
+
+namespace ChainSafe.GamingSdk.Editor
+{
+    /// <summary>
+    /// Make connection providers read-write by moving it from packages into Assets/Resources.
+    /// </summary>
+    [InitializeOnLoad]
+    public class ReadWriteProvider
```

Comment:
Don't think this makes much sense imo. Because, for people importing the package, the files would be in the package cache. Which cannot be deleted permanently.

Reply:
Why is it important to make them read/write?

Reply:
because we'll need to edit the values on the scriptable objects and if they're in packages they can't be edited, I guess I could change the name to `EditableConnectionProviders`, I think we should keep this script because when we add a new package like HyperPlay it'll automatically add the provider

Reply:
Actually, what I meant is, that we shouldn't add ConnectionProviders to the Resources folder in the package itself. But rather, let them sit in the root or in a folder named ConnectionProviderData, and transfer them from there to the resources inside of the Assets folder, because otherwise you'd get duplicate providers..

Reply:
I actually had a dilemma about this because providers are like configs and if that's the case shouldn't we make user configure/create them? I suppose we can have a button under providers in connection handler like setup/configure for each type of provider to make it easier but configuring should be for dev imo for example we need the user to setup their own wallet connect project on their dashboard and get the id to reference in the config same for web3auth so shouldn't we let users configure their own providers, let me know what you think

Reply:
Actually, yes, that is the best solution, and kind of like a standard procedure. So less hassle for us.